src/content/ui-api/kubectl-gs/template-cluster.md

```diff
@@ -87,7 +98,32 @@ It supports the following flags:
 
 ### AWS specific
 
-- `--external-snat` - AWS CNI configuration to disable (is enabled by default) the [external source network address translation](https://docs.aws.amazon.com/eks/latest/userguide/external-snat.html). Only versions *11.3.1+ support this feature.
+- `--external-snat` - AWS CNI configuration to disable (is enabled by default) the [external source network address translation](https://docs.aws.amazon.com/eks/latest/userguide/external-snat.html).
+
+### OpenStack specific
+
+- `--cluster-version` (optional) - Version of `cluster-openstack` helm chart to use. If not provided, the latest version will be used.
+- `--default-apps-version` (optional) - Version of `default-apps-openstack` helm chart to use. If not provided, the latest version will be used.
+- `--cloud` - Name of the cloud in the `cloud-config` secret. This is almost always "openstack".
+- `--cloud-config` - Name of the `cloud-config` secret which defines the credentials for the OpenStack project in which the cluster should be created. This must be created in the organization namespace before creating a cluster.
+- `--dns-nameservers` (optional) - A list of DNS nameservers to be used to resolve external names.
+- `--external-network-id` - UUID of the external network to be used. Only required if multiple external networks are available.
+- `--failure-domain` - Failure domain of worker nodes.
```

Comment:
There is some discrepancy between `--failure-domain` and `--worker-failure-domain`